mlflow/store/db/utils.py

```diff
@@ -83,7 +83,7 @@ def make_managed_session():
             raise
         except Exception as e:
             session.rollback()
-            raise MlflowException(message=e, error_code=INTERNAL_ERROR)
+            raise MlflowException(message=str(e), error_code=INTERNAL_ERROR)
```

Comment:
The `e` might be database `IntegrityError` which cannot be dumped as JSON, this might cause error such the one reported by https://github.com/mlflow/mlflow/issues/5704